src/components/MapView/SimpleDropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormControl, MenuItem, Select } from '@material-ui/core';
```

Comment:
I definitely agree.  I intentionally built it in order to be reused across the application :-) I looked and didn't see a great folder home for it, so I just put it an existing folder that made the most sense to me.  Would perhaps creating a `src/components/Common` folder make sense to you?  It would be a folder for Common components like this that could be shared across the application.  `src/components/Shared` is also an option.

Also, definitely open to your suggestions for other folder names or paths.

Reply:
I moved SimpleDropdown to a new Common folder via https://github.com/WFP-VAM/prism-frontend/pull/431/commits/c6ab13d58924846e69d8204edd7cdd9c6c410c30